doc/_sphinx/theme/doctools.js

```diff
@@ -21,11 +21,18 @@ const _ready = (callback) => {
   }
 };
 
+const BLACKLISTED_KEY_CONTROL_ELEMENTS = new Set([
+  "TEXTAREA",
+  "INPUT",
+  "SELECT",
+  "BUTTON",
+]);
+
 /**
  * highlight a given string on a node by wrapping it in
  * span elements with the given class name.
  */
-const _highlight = (node, addItems, text, className, index) => {
+const _highlight_flame = (node, addItems, text, className, index) => {
```

Comment:
I guess this should be `_highlightFlame` now, to keep everything consistent :)

Reply:
Done.  Now we can't parse any of the modified variables by theme name, so if that situation arises down the road until a full theme overhaul, it will just need to be considered then.